src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs

```diff
@@ -96,17 +97,28 @@ public FrameRenderer(Context context, IPropertyMapper mapper, CommandMapper comm
 			}
 		}
 
-		Size IViewHandler.GetDesiredSize(double widthMeasureSpec, double heightMeasureSpec)
+		Size IViewHandler.GetDesiredSize(double widthConstraint, double heightConstraint)
 		{
-			double minWidth = 20;
-			if (Primitives.Dimension.IsExplicitSet(widthMeasureSpec) && !double.IsInfinity(widthMeasureSpec))
-				minWidth = widthMeasureSpec;
+			var virtualView = (this as IViewHandler)?.VirtualView;
+			if (virtualView is null)
+			{
```

Comment:
Do we have a way to test this also?

Reply:
There's a test in this PR.

Reply:
Unless you mean the `null` scenario?

Reply:
Yeah when would it get Size zero?

Reply:
If someone disconnect the handler in a non-UI thread, or something weird like that. The point is that if the VirtualView is null, then having any size at all doesn't make sense.